src/components/Border/Border.css

```diff
@@ -0,0 +1,25 @@
+#border-container {
+    background-color: rgb(240, 240, 240);
```

Comment:
another thing it is better to use hex colors instead of rgb 
[rgb to hex](https://www.rgbtohex.net/)